src/AppInstallerCommonCore/Errors.cpp

```diff
@@ -154,6 +154,8 @@ namespace AppInstaller
                 return "Failed to open the source.";
             case APPINSTALLER_CLI_ERROR_SOURCE_AGREEMENTS_NOT_ACCEPTED:
                 return "Source agreements were not agreed to";
+            case APPINSTALLER_CLI_ERROR_CUSTOMHEADER_EXCEEDS_MAXLENGTH:

```

Comment:
>APPINSTALLER_CLI_ERROR_CUSTOMHEADER_EXCEEDS_MAXLENGTH [](http://example.com/codeflow?start=17&length=53)

nit: For completeness, update the e2e tests errors table too since you'll have to fix the spelling already